test/client-unit/filters/percentage.spec.js

```diff
@@ -0,0 +1,31 @@
+/* global inject, expect */
+describe('Percentage Filter', () => {
+
+  let percentage;
+
+  beforeEach(module('pascalprecht.translate', 'bhima.filters'));
+
+  beforeEach(inject(($filter) => {
+    percentage = $filter('percentage');
+  }));
+
+  it('returns an empty string for undefined', () => {
+    expect(percentage()).to.equal('');
+    expect(percentage(undefined)).to.equal('');
+  });
+
+  it('returns the value if the value is not a number', () => {
```

Comment:
Is there any case where the percentage filter can receive a string or an object ? Since they are not numeric values why not to just return an empty string

Reply:
I can't remember who wrote the percentage filter, but I believe the idea probably was to let a developer know when they have made a mistake by printing out the object.  I'm not certain though.

What do you think?  Should I update the behavior of the percentage filter?

Reply:
It will be really good if you can do that

Reply:
It has been done.